pip/commands/install.py

```diff
@@ -369,3 +356,31 @@ def run(self, options, args):
                 )
             shutil.rmtree(temp_target_dir)
         return requirement_set
+
+
+class InstallCommand(InstallBase):
+    """
+    Install packages from:
+
+    - PyPI (and other indexes) using requirement specifiers.
+    - VCS project urls.
+    - Local project directories.
+    - Local or remote source archives.
+
+    pip also supports installing from "requirements files", which provide
+    an easy way to specify a whole environment to be installed.
+    """
+    name = 'install'
+    summary = 'Install packages.'
+    upgrade_option = True
+    upgrade_direct = False
```

Comment:
this may be bikeshedding, but this "upgrade_direct" phrase is pretty meaningless to me.

I'd prefer "upgrade_recursive", which will be True for the InstallCommand, and False for the UpgradeCommand